sys-devel/llvm/patches/llvm18-18.1.7.patchset

```diff
@@ -737,5 +737,34 @@ index d3ba115..c9a272e 100644
      config.unsupported = True
  
 -- 
-2.45.2
+2.48.1
+
+
+From b1120b9cd36fc0e3a9581041d484044d870fbed7 Mon Sep 17 00:00:00 2001
+From: Luc Schrijvers <begasus@gmail.com>
+Date: Thu, 17 Apr 2025 13:43:33 +0200
+Subject: Fix missing PATH for stddef
+
+
+diff --git a/clang/lib/Driver/ToolChains/Haiku.cpp b/clang/lib/Driver/ToolChains/Haiku.cpp
```

Comment:
@korli this should do it I guess, if OK I'll apply this to the source for LLVM19 and LLVM20 also.

Reply:
Should be good I think